index.js

```diff
@@ -3638,15 +3638,21 @@
   //.
   //. > S.get(S.is(Number), 'x', {})
   //. Nothing
+  //.
```

Comment:
> 'purple' is meant to satisfy the Accessible constraint

Yeah that's what I'm saying.  That behaviour is logical to me but could perhaps be illustrated in the examples further.